cv/qc-cv.obo

```diff
@@ -18,1967 +18,341 @@ import: https://raw.githubusercontent.com/ISA-tools/stato/dev/releases/1.4/stato
 ontology: ms
 
 remark: check for terms moved to PSI-MS-CV
-
-[Term]
-id: MS:4000065
-name: Precursor median m/z for IDs
-def: "Median m/z value for all identified peptides (unique ions) after FDR." [PSI:QC]
-is_a: MS:4000003 ! single value
-is_a: MS:4000009 ! ID based
-is_a: MS:4000023 ! MS1 metric
-is_a: MS:4000025 ! ion source metric
-relationship: has_relation MS:4000013 ! single run based
-
-[Term]
-id: MS:4000066
-name: Fraction of MS2 identified at different MS1 quantiles
-def: "Fraction of total MS2 scans identified after FDR in the respective quantile of peptides sorted by MS1 maximum intensity." [PSI:QC]
-is_a: MS:4000004 ! n-tuple
-is_a: MS:4000009 ! ID based
-is_a: MS:4000023 ! MS1 metric
-is_a: MS:4000024 ! MS2 metric
-relationship: has_relation MS:4000013 ! single run based
-
-[Term]
-id: MS:4000067
-name: Total ion current chromatogram table
-def: "The total ion current chromatogram. The first column contains the retention time values in second, the second column the corresponding relative intensities." [PSI:QC]
-is_a: MS:4000006 ! table
-is_a: MS:4000010 ! ID free
-is_a: MS:4000022 ! chromatogram metric
-relationship: has_relation MS:1000235 ! total ion current chromatogram
-relationship: has_relation MS:4000013 ! single run based
-relationship: has_column: MS:4000108 ! Retention time
-relationship: has_column: MS:4000109 ! Relative intensity
+remark: and terms not in this batch
+remark: CV migration batch no. 2
 
 [Term]
 id: MS:4000068
-name: Ambient humidity
-def: "The ambient relative humidity in percent (0,100) over one or more time points. The first column contains is the ambient humidity value(s). The second contains the respective retention time. A negative RT implies that the exact time of measurement is unknown." [PSI:QC]
-is_a: MS:4000006 ! table
-is_a: MS:4000010 ! ID free
-is_a: MS:4000026 ! environment metric
-relationship: has_relation MS:4000013 ! single run based
-relationship: has_column: MS:4000108 ! Retention time
-relationship: has_column: MS:4000260 ! Percent
+name: number of PSM
```

Comment:
In consistancy with the next item, add "before FDR".
I would rather suggest even to call it something like "number of PSMs (unfiltered)" to make it more general (also in the following term for spectra)

Reply:
I would add it should be `number of PSM**s**` ?!
and while unfiltered as a word is nice, this may lead to semantic issues, because sometimes you do have prefiltered results, e.g. when using XTandem's `maximum valid e-value` filter.
Also these sequences are prefiltered by (tryptic) specificity etc.. (but this is splitting hairs now :P)